index.html

```diff
@@ -0,0 +1,1402 @@
+              </a>
+              <a href="#">
+                <h2 class="card-title">Charles Barkley Blasted For Supporting Players Who Stand For Anthem</h2>
+              </a>
+            </div>
+          </div>
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/lebron-james-kawhi-leonard-los-angeles-lakers.webp" alt="kawhi">
+              </a>
+              <a href="#">
+                <h2 class="card-title">
+                  NBA Schedule: Clippers vs. Lakers Start Time, TV Channel, Live Stream</h2>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row w-100 d-inline-flex border-top">
+        <div class="col-12 col-md-8 d-inline-block my-4">
+          <h3>SUBSCRIBE</h3>
+          <div>
+            <img class="float-left mr-5" height="200" src="./img/cover-july-24-2020.jpg" alt="cover">
+            <div class="pt-5">
+              <h4>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+              <p>All Access + Weekly Delivery.
+                Choose A Membership That's Perfect for You!</p>
+              <button class="btn-secondary" type="button">SUBSCRIBE</button>
+            </div>
+          </div>
+        </div>
+        <div class="col-12 col-md-4 my-4">
+          <h3>NEWSLETTER</h3>
+          <div class="d-inline-flex h-100">
+            <div class="mr-2">
+              <img width="200" height="200" src="./img/130387482-newsletter-subscription-button-on-mobile-phone-screen-business-marketing-concept-.jpg" alt="">
+            </div>
+            <div>
+              <h4>The Starting 5</h4>
+              <p>
+                See why nearly a quarter of a million subscribers begin their day with the Starting 5.
+              </p>
+              <div class="input-group my-2">
+                <input class="form-control" type="email" placeholder="Email address">
+              </div>
+              <button class="btn-secondary w-100" type="button">FREE SIGN UP <i class="fas fa-angle-right"></i></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer class="container-fluid border-top width">
+    <!--Footer Header-->
+    <div class="row justify-content-between py-4">
+      <div class="col-md-7 col-12 d-flex mb-3">
+        <img class="img-fluid" src="./img/logo-red.png" alt="logo-red" width="200">
+        <p>&copy; 2020 NEWSWEEK</p>
+      </div>
+      <div class="col-md-5 col-12 d-flex justify-content-md-end">
+          <div class="mr-2">
+            <i class="fab fa-facebook-square fa-2x"></i>
+          </div>
+          <div class="mr-2">
+            <i class="fab fa-twitter fa-2x"></i>
+          </div>
+          <div class="mr-2">
```

Comment:
- [ ] copyright alignment on the mobile screen is different. Also, pay attention to the font styling.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89236167-3b19aa00-d5c6-11ea-908b-722d9781ebcf.png) | ![image](https://user-images.githubusercontent.com/61479648/89236195-4d93e380-d5c6-11ea-8bad-0b025b6bad2b.png) |